tests/unit/proxy/abstract_proxy_tests.py

```diff
@@ -42,6 +59,183 @@ def tearDown(self) -> None:
     def get_proxy(self) -> T:
         pass
 
+    @abstractmethod
+    def get_relationship(self, *, node_type1: str, node_key1: str, node_type2: str,
+                         node_key2: str) -> List[Any]:
+        pass
+
+    def test_rt_table(self) -> None:
+        """
+        it'd be nice to check that the result could be deserialized as a client of the metadata_service would
+        """
+        # TODO: check fixtures
+        expected = Fixtures.next_table()
+        expected.description = '"hello!" said no one'
+        expected.tags.sort()
+
+        self.get_proxy().put_table(table=expected)
+        actual: Table = self.get_proxy().get_table(table_uri=checkNotNone(expected.key))
+        actual.last_updated_timestamp = None
+        actual.tags.sort()
+
+        self.assertEqual(expected, actual)
+
+    def test_rt_table_with_owner_user_not_application(self) -> None:
```

Comment:
This might also be a square-ism. I'm not sure other Amundsen users have a concept of application vs user owners of a table

Reply:
right! Will rename